<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMainWindowClass</class>
 <widget class="QMainWindow" name="QtMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="MyGraphicsView" name="graphicsView">
      <property name="statusTip">
       <string>Graphics view</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="foregroundBrush">
       <brush brushstyle="NoBrush">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_Copy"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menuEinf_gen">
    <property name="title">
     <string>Einfügen</string>
    </property>
    <addaction name="action_Rect"/>
    <addaction name="action_Picture"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEinf_gen"/>
   <addaction name="menuZoom"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="action_Copy"/>
   <addaction name="action_Cut"/>
   <addaction name="action_Paste"/>
   <addaction name="action_Rect"/>
   <addaction name="action_Picture"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/new.png</normaloff>:/QtMainWindow/images/new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Neu</string>
   </property>
   <property name="toolTip">
    <string>Neue Datei</string>
   </property>
   <property name="statusTip">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/open.png</normaloff>:/QtMainWindow/images/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Öffnen</string>
   </property>
   <property name="toolTip">
    <string>Öffnen einer Datei</string>
   </property>
   <property name="statusTip">
    <string>Open a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/copy.png</normaloff>:/QtMainWindow/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Kopieren</string>
   </property>
   <property name="statusTip">
    <string>Kopieren</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/save.png</normaloff>:/QtMainWindow/images/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Speichern</string>
   </property>
   <property name="statusTip">
    <string>In Dokument speichern</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/cut.png</normaloff>:/QtMainWindow/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ausschneiden</string>
   </property>
   <property name="statusTip">
    <string>Auschneiden und Kopieren</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/paste.png</normaloff>:/QtMainWindow/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Ein&amp;fügen</string>
   </property>
   <property name="statusTip">
    <string>Aus Zwischenspeicher einfügen</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_Rect">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/ico_draw_rect.png</normaloff>:/QtMainWindow/images/ico_draw_rect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rechteck</string>
   </property>
   <property name="statusTip">
    <string>Male ein Rechteck</string>
   </property>
  </action>
  <action name="action_Picture">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/canvas.png</normaloff>:/QtMainWindow/images/canvas.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bild</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/ico_layer_add.png</normaloff>:/QtMainWindow/images/ico_layer_add.png</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="statusTip">
    <string>Vergrößern der Ansicht</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="qtmainwindow.qrc">
     <normaloff>:/QtMainWindow/images/ico_layer_erase.png</normaloff>:/QtMainWindow/images/ico_layer_erase.png</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="statusTip">
    <string>Verkleinern der Ansicht</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>mygraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qtmainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
